src/Graphics/Circle.h

```diff
@@ -0,0 +1,18 @@
+#ifndef _CIRCLE_H_
+#define _CIRCLE_H_
+
+#include "Shape.h"
+
+class Circle : public Shape {
```

Comment:
I'm curious what these classes add in functionality beyond the graphics library methods we're already using to draw shapes? There's quite a few changes, so I'm trying to wrap my head around it.

Reply:
This is mostly an organizational thing as they contain all their own coordinates / size / color etc.. so we don't have to store shape data as global values which we reference every time we want to draw a particular graphic

Reply:
also as an interface, it's nice that we only have to define a shape and all it's properties once.  it just holds onto those properties so we can call draw() or clear() without having to re-specify what we're drawing